### DISCUSSION TITLE
Read only

### DISCUSSION THREAD

**Discussion by Scooby64** [filebrowser/filebrowser#1831] [Q&A]
Hello.

I would like to know if you can put a folder / file in read only

thanks in advance

**Comment by ramiresviana** [comment]
Filebrowser does not provide this functionality, most similar to this is the sharing feature. You can also use file system permissions to have read-only folders/files.

**Comment by Scooby64** [comment]
Thank you for this quick response.
I tried with the chmod command to put a folder in "444" or "555".
Connected to the command line in "admin" I could no longer delete it. So I thought that was good.
But in FileBrowser I could delete it...

**Reply by ramiresviana** [reply]
Filebrowser has the same privileges as the user that runs it.
